pages/components/Input/input.css

```diff
@@ -0,0 +1,46 @@
+.inp {
```

Comment:
You could name it ```input```. It would give more context that where does it belong to?. (Just a suggestion)

Reply:
Hey, Aditya thanks for the feedback I have changed the cursor to pointer wherever necessary and I have renamed the class too.